driver/razermouse_driver.h

```diff
@@ -90,6 +90,7 @@
 #define USB_DEVICE_ID_RAZER_HYPERPOLLING_WIRELESS_DONGLE 0x00B3
 #define USB_DEVICE_ID_RAZER_DEATHADDER_V3_PRO_WIRED 0x00B6
 #define USB_DEVICE_ID_RAZER_DEATHADDER_V3_PRO_WIRELESS 0x00B7
+#define USB_DEVICE_ID_RAZER_NAGA_V2_HYPERSPEED_RECEIVER 0x00B4
```

Comment:
move this up just below `USB_DEVICE_ID_RAZER_HYPERPOLLING_WIRELESS_DONGLE` so the list is sorted by value